spec/mongoid/attributes/nested_spec.rb

```diff
@@ -14,20 +14,31 @@
 
       before do
         Person.accepts_nested_attributes_for :favorites
+        Person.accepts_nested_attributes_for :children
```

Comment:
This is one of the mutations added in this diff.

Instead of mutating `Person` here, add another model and configure it with `children` without autosave and put a note on top of it that it's owned by the `nested_spec` test and not to be used by anyone else. Add another model configured with `autosave: false` with the same note. Use those models in the tests being added in this PR.

Reply:
The model is already being mutated in the exact same way, just on a different field. Again, I don't want to add a bunch of random models ad-hoc every time we touch a test that uses a pattern like this. If we want to change this pattern, we should tackle it at once in a coherent unit of work rather than just piggybacking it piecemeal on top of unrelated tickets.

Reply:
Global mutations fail the test suite in jruby as well as mri (less frequently) and fail when parallelized. They are a constant drain on engineer productivity because they cause failures in unrelated parts of the test suite, as you said yourself in this PR. I am happy to discuss other approaches to testing but global mutations are not good.

Reply:
I understand that, and I agree. However, fixing these two individual mutations won't fix the parallel tests though given that it's done throughout this file (and presumably elsewhere in the test suite). That's why I think we should schedule work to fix all of them at once rather than just doing piecemeal work on tickets that aren't related.